specification/changes/data-plane/Microsoft.Resources/preview/2022-01-01/changes.json

```diff
@@ -0,0 +1,421 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-01-01",
+    "title": "ChangesClient",
+    "description": "The Resource Changes Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/{resourceProviderNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Resources/changes": {
+      "get": {
+        "tags": [
+          "ListChanges"
+        ],
+        "operationId": "ChangeResources_List",
+        "x-ms-examples": {
+          "ListChanges": {
+            "$ref": "./examples/ListChanges.json"
+          }
+        },
+        "description": "Obtains a list of change resources from the past 14 days for the target resource",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceProviderNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/top"
+          },
+          {
+            "$ref": "#/parameters/skipToken"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/ResourceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true,
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/{resourceProviderNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Resources/changes/{changeResourceId}": {
```

Comment:
>Resources [](http://example.com/codeflow?start=150&length=9)

Is this related to the Microsoft.Resources namespace from : [https://github.com/Azure/azure-rest-api-specs/tree/88e7838a09868a51de3894114355c75929847a46/specification/resources/resource-manager/Microsoft.Resources](https://github.com/Azure/azure-rest-api-specs/tree/88e7838a09868a51de3894114355c75929847a46/specification/resources/resource-manager/Microsoft.Resources)

Reply:
Please rename if this is different